src/server/test/web/csvPipelineTest.js

```diff
@@ -472,6 +476,13 @@ const testCases = {
 		fileName: ['pipe123AInputMeter.csv', 'pipe123BInput.csv'],
 		responseCode: [200, 200],
 		responseString: ['<h1>SUCCESS</h1>Successfully inserted the meters.', '<h1>SUCCESS</h1><h2>It looks like the insert of the readings was a success.</h2><h3>However, note that the processing of the readings returned these warning(s):</h3><br>For meter pipe123: Warning parsing Reading #2. Reading value gives 30 with warning message:<br>The reading start time is shifted and within the DST shift so it is possible that the crossing to standard time was missed and readings overlap. The current reading startTime is not after the previous reading\'s end time. Note this is treated only as a warning since readings may be sent out of order.<br>There is a gap in time between this reading and the previous reading that exceeds the allowed amount of 0 seconds.<br>For reading #2 on meter pipe123 in pipeline: previous reading has value 15 start time 2022-11-06T01:25:00Z end time 2022-11-06T01:55:00Z and current reading has value 30 start time 2022-11-06T01:30:00Z end time 2022-11-06T02:00:00Z with timeSort increasing; duplications 1; cumulative false; cumulativeReset false; cumulativeResetStart 00:00:00; cumulativeResetEnd 23:59:59.999999; lengthGap 0; lengthVariation 0; onlyEndTime false<br>']
+	},
+	pipe1: {
```

Comment:
A test key should never be reused. It is a unique identifier. This should be a new test number - maybe pipe130? The same applies for the other test changed.

Reply:
Got it. I will change these.